src/components/ActionButton/ActionButton.tsx

```diff
@@ -119,6 +119,7 @@ const ActionButton = ({
   return (
     <button
       className={buttonClasses}
+      disabled={disabled}
```

Comment:
This needs to be explicitly defined because it's an explicitly defined `ActionButton` prop, i.e. it's not spread in `...buttonProps`